src/ui/profile_preferences.ui

```diff
@@ -1264,11 +1284,14 @@
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="pushButton_lBlack">
+            <property name="toolTip">
+             <string>ANSI Color Number 8</string>
```

Comment:
:+1:

Reply:
shouldn't "color" and "number" be lower case though?

Reply:
Well, they are specific rather than general cases "Color Number" rather than "color number" but perhaps you are right... 😐

Reply:
...will change.